### PR TITLE
USB Devices: Make requirement for USB qube more visible & remove redundant sentence

### PR DESCRIPTION
**Pull request by WillyPillow**
I was playing with `qvm-usb` a few days ago, only to find, after some failures and head-scratching, that a USB qube is required.

Issues such as QubesOS/qubes-issues#4345 also show the confusion of other users, so it should make sense to move the requirement to the beginning where it is more visible.

Also, I find that the sentence "`qubes-usb-proxy` should be installed..." to only reiterate what was being said in the previous paragraph, so I opt to remove it.

### CODE REVIEW

File: user/common-tasks/usb-devices.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ redirect_from:
 
 If you are looking to handle USB *storage* devices (thumbdrives or USB-drives), please have a look at the [block device] page.
 
+**Note:** you cannot pass through devices from dom0 (in other words: a [USB qube][USB-qube howto] is required).
```
Comment (andrewdavidwong) [inline user/common-tasks/usb-devices.md:15]: I think this sentence is very confusing when removed from its previous context (and perhaps it was already confusing before). Seeing it near the top of the page like this, I'm not even sure how to interpret it.

Comment (andrewdavidwong) [reply user/common-tasks/usb-devices.md:15]: If I were a user reading this page for the first time, after reading this sentence, I would probably wonder:
 - Pass through to where?
 - Why can't I pass through devices from dom0 if that's where they are? How else am I supposed to get them out?
 - A USB qube is required for what?

Comment (WillyPillow) [reply user/common-tasks/usb-devices.md:15]: Perhaps it is less confusing to rephrase the sentence to something like the following?

> Attaching USB devices to VMs requires a [USB qube](USB-qube howto).

One can also move the sentence to the "Creating And Using a USB qube" section, though again the visibility may not be as high.

(For the record, my original justification for placing the sentence there untouched was that, as the term "USB passthrough" was used quite frequently in the first section and throughout the page, I ended up assuming the concept to be prior knowledge. I understand the concerns though.)

Comment (andrewdavidwong) [reply user/common-tasks/usb-devices.md:15]: I think your new proposed sentence is much clearer and simpler.

Comment (WillyPillow) [reply user/common-tasks/usb-devices.md]: The updated version is pushed.

**Pull request merged**